asttokens/mark_tokens.py

```diff
@@ -279,3 +280,9 @@ def visit_assignname(self, node, first_token, last_token):
       colon = self._code.find_token(last_token, token.OP, ':')
       first_token = last_token = self._code.prev_token(colon)
     return (first_token, last_token)
+
+  if six.PY2:
```

Comment:
Oh, that's certainly the better way to do the check haha. Plus, the `import sys` above is no longer needed :D